bootstrap-app/src/cmr/bootstrap/data/virtual_products.clj

```diff
@@ -107,7 +107,8 @@
   [system source-collection source-granule virtual-collection]
   (let [virtual-granule-ur (svm/generate-granule-ur (:provider-id source-collection)
                                                     (:short-name source-collection)
-                                                    (-> virtual-collection :extra-fields :short-name)
+                                                    {:short-name (-> virtual-collection :extra-fields :short-name)
+                                                     :version-id (-> virtual-collection :extra-fields :version-id)}
```

Comment:
I like this, but why not use `get-in`?

Reply:
Ancient code, I just copied the style in moment.  I'll replace, thanks.